x-pack/plugins/security_solution/public/management/components/effected_policy_select/effected_policy_select.tsx

```diff
@@ -206,36 +209,43 @@ export const EffectedPolicySelect = memo<EffectedPolicySelectProps>(
               </p>
             </EuiText>
           </EuiFlexItem>
-          <EuiFlexItem grow={1}>
-            <EuiFormRow fullWidth>
-              <EuiButtonGroup
-                legend="Global Policy Toggle"
-                options={toggleGlobal}
-                idSelected={isGlobal ? 'globalPolicy' : 'perPolicy'}
-                onChange={handleGlobalButtonChange}
-                color="primary"
-                isFullWidth
-              />
-            </EuiFormRow>
-          </EuiFlexItem>
+          {!arePoliciesLoading && (
```

Comment:
Just to make sure: the entire toggle is hidden if we don't have the policies loaded yet. but I wonder if we should only show the loader if the current selection is `Policy` and not `Global`. Thoughts?

Reply:
Yeah, that make sense too